.github/workflows/ci-pipeline.yaml

```diff
@@ -95,7 +81,7 @@ jobs:
           if [[ -d "${{ matrix.container }}" ]]; then
             name="$(echo "${{ matrix.container }}" | awk -F '/' '{print $2}')"
             tag=""
-            if [[ "${{ github.actor }}" == "bitnami-bot" ]]; then
+            if [[ "${{ github.event.pull_request.user.login }}" == "bitnami-bot" ]]; then
```

Comment:
I don't want to be squeamish but I'd use the ' instead of " for the _bitnami-bot_ string like you did before.